00:00:00.001731 main     VirtualBox COM Server 6.1.34 r150636 win.amd64 (Mar 23 2022 00:36:21) release log
00:00:00.001733 main     Log opened 2022-08-14T21:52:00.368154000Z
00:00:00.001733 main     Build Type: release
00:00:00.001737 main     OS Product: Windows 11
00:00:00.001737 main     OS Release: 10.0.22000
00:00:00.001738 main     OS Service Pack: 
00:00:00.037651 main     DMI Product Name: ROG Zephyrus G14 GA402RJ_GA402RJ
00:00:00.044113 main     DMI Product Version: 1.0       
00:00:00.044130 main     Firmware type: UEFI
00:00:00.044390 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.044407 main     Host RAM: 15612MB (15.2GB) total, 7961MB (7.7GB) available
00:00:00.044411 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.044412 main     Process ID: 14656
00:00:00.044412 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.085067          VirtualBox: object creation starts
00:00:00.085223          Home directory: 'C:\Users\amrit\.VirtualBox'
00:00:00.085332          Installed Drivers:
00:00:00.088044            C:\WINDOWS\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.34.636)
00:00:00.088693            C:\WINDOWS\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.34.636)
00:00:00.088987            C:\WINDOWS\system32\DRIVERS\VBoxSup.sys (Version: 6.1.34.636)
00:00:00.089920            C:\WINDOWS\system32\DRIVERS\VBoxNetAdp6.sys (Version: 6.1.34.636)
00:00:00.100638          Loading settings file "C:\Users\amrit\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.102014          usbLibGetDevices: Starting USB device enumeration
00:00:00.171022          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:00:00.235730          usbLibGetDevices: Found 4 USB devices, 0 captured
00:00:00.235795          USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56eb)
00:00:00.315970          HostDnsMonitor: initializing
00:00:00.323377          HostDnsMonitor: updating information
00:00:00.323395          HostDnsMonitor: old information
00:00:00.323395            no server entries
00:00:00.323396            no domain set
00:00:00.323396            no search string entries
00:00:00.323396          HostDnsMonitor: new information
00:00:00.323397            server 1: 64.71.255.204
00:00:00.323397            server 2: 64.71.255.198
00:00:00.323398            domain: phub.net.cable.rogers.com
00:00:00.323398            no search string entries
00:00:00.350447          VD: VDInit finished with VINF_SUCCESS
00:00:00.354779          Loading settings file "C:\Users\amrit\VirtualBox VMs\Ubuntu\Ubuntu.vbox" with version "1.16-windows"
00:00:00.355403          VirtualBox: object created
00:00:18.195897          Saving settings file "C:\Users\amrit\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:18.203500          Saving settings file "C:\Users\amrit\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:18.983654          Launched VM: 68295280 pid: 17704 (0x4528) frontend: GUI/Qt name: Ubuntu
00:00:22.263266          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:37:16.845309 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:37:16.861316 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2524570b&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:37:16.908887 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:37:16.945190 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
00:52:14.060085 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:52:14.077679 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2524570b&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:52:14.126833 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:52:14.164475 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
00:52:17.511982 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:52:17.581841 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:52:17.619974 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
00:52:20.440629 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:52:20.510876 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:52:20.548132 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
00:52:32.515073 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:52:32.532009 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2524570b&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:52:32.581905 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
00:52:32.618716 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
06:04:14.870228 dns-monitor HostDnsMonitor: updating information
06:04:14.870286 dns-monitor HostDnsMonitor: unchanged
06:04:16.116178 USBPROXY usbLibGetDevices: Starting USB device enumeration
06:04:16.227580 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
06:04:20.474861 USBPROXY usbLibGetDevices: Starting USB device enumeration
06:04:20.542925 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
06:04:20.579235 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
06:04:20.579293 USBPROXY USB: Unknown USB device detected (idVendor: 0x13d3, idProduct: 0x56eb)
06:04:27.878495 dns-monitor HostDnsMonitor: updating information
06:04:27.878517 dns-monitor HostDnsMonitor: unchanged
06:04:53.089616 USBPROXY usbLibGetDevices: Starting USB device enumeration
06:04:53.158012 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
06:04:53.192947 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
06:05:03.079147 USBPROXY usbLibGetDevices: Starting USB device enumeration
06:05:03.095647 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2524570b&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
06:05:03.144010 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
06:05:03.179651 USBPROXY usbLibGetDevices: Found 4 USB devices, 0 captured
06:20:15.691170 USBPROXY usbLibGetDevices: Starting USB device enumeration
06:20:15.707025 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2524570b&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
06:20:15.754998 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&39b8775f&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 3
06:20:15.779546 USBPROXY usbLibGetDevices: Found 5 USB devices, 0 captured
06:24:02.597456          ERROR [COM]: aRC=VBOX_E_IPRT_ERROR (0x80bb0005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={SessionMachine} aText={Saved screenshot data is not available (VERR_NOT_SUPPORTED)}, preserve=false aResultDetail=-37
06:24:04.111558          Saving settings file "C:\Users\amrit\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
06:24:04.119123          Saving settings file "C:\Users\amrit\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
06:24:04.122101          Saving settings file "C:\Users\amrit\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
06:24:12.376191          Saving settings file "C:\Users\amrit\VirtualBox VMs\Ubuntu\Ubuntu.vbox" with version "1.16-windows"
06:24:12.885897          Saving settings file "C:\Users\amrit\VirtualBox VMs\Ubuntu\Ubuntu.vbox" with version "1.16-windows"
06:24:12.891741          Saving settings file "C:\Users\amrit\VirtualBox VMs\Ubuntu\Ubuntu.vbox" with version "1.16-windows"
06:24:18.409595 main     VirtualBox: object deletion starts
06:24:18.410091 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\amrit\VirtualBox VMs\Ubuntu\Ubuntu.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
06:24:18.411178 main     HostDnsMonitor: shutting down ...
06:24:18.411225 main     HostDnsMonitor: shut down
06:24:18.412358 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
06:24:18.412489 main     VirtualBox: object deleted
